packages/airnode-node/src/types.ts

```diff
@@ -209,24 +210,46 @@ export interface AuthorizationByRequestId {
   readonly [requestId: string]: boolean;
 }
 
-export interface ApiCallResponse {
-  readonly value?: string | boolean;
-  readonly errorMessage?: string;
+export type ApiCallResponse = ApiCallSuccessResponse | ApiCallErrorResponse;
+
+export interface ApiCallSuccessResponse {
+  success: true;
```

Comment:
Do we want to make these interfaces readonly like the others?

Reply:
I actually dislike that because:
1. It creates unnecessary friction in tests and when using local muration
2. Provides zero benefit since our code is largely functional (in the important places)

and I have to say I dislike it kinda in general :D